mobile/function/route_list_functions.js

```diff
@@ -32,8 +33,11 @@ export const GetRouteInformation = (apiCallParameter) => {
         const json = await response.json();
 
         setData(json);
+        console.log(json);
```

Comment:
ook console.log weg halen tenzij je die gebruikt die er met een reden is natuurlijk